ConsumingEvents.md

```diff
@@ -59,15 +55,23 @@ The receiver code creates an *EventHubClient* from a given connecting string
 		
     EventHubClient ehClient = EventHubClient.createFromConnectionStringSync(connStr.toString());
 ```           
+### Azure IoT Hub
```

Comment:
The example connection string below contains a number of items that the user would need to replace which are not marked as such. The linked doc does a good job of covering this already, so can you remove the example and move this section to the end of the doc under Connection Strings?